### DISCUSSION TITLE
SSH Jumphost - Surprise about SSH profile inheritance

### DISCUSSION THREAD

**Discussion by RummsMurmel** [Eugeny/tabby#8330] [Q&A]
Hello all, 

I am a bit puzzled about how to create a SSH jumphost for the default SSH profile.

**What I am trying to do (my goal):**
Since I have more than 100 hosts to manage, I don't want to build a profile for each SSH connection. So I want to connect to various hosts directly by entering FQDN or IP in the top search box instead of creating a profile for each host in advance. I need to use a Jump-Host, to connect to my servers. So I want the default SSH profile to use a jumphost. 

**What I did:**
1. created the jump host profile. For this I created a new profile with the template "SSH-Connection".  I selected "Direct" as the connection method and then configured the FQDN + my username + agent auth. A test to connect works.

2. in "Advanced" Settings I changed the default behavior for the SSH profile to:
- Hostname: empty, since I want to connect dynamically via FQDN/IP.
- Connection method: Jump server
- Jump server: Here I have chosen my profile created in step 1.

**My problem:**
By changing the default SSH settings in step 2, my profile created in step 1 was changed to "Jump-Host" - but my Jump-Host should be addressed "Direct". So neither the connection to the jump host nor to any other server works. 
Also changing the Jump-Host profile to "Direct" afterwards does not work - Tabby ignores the change every time and changes it back to "Jump-Host".

**Expected behavior:**
I would have expected that after changing the default SSH profile in step 2, I could manually change the connection method for my dedicated jump host profile afterwards. 

In the release notes I see for version "Alpha 154" the hint: 
"Fixed jump hosts not inheriting default profile settings". 

So apparently the behavior was even wanted once - but I don't understand how to get my application scenario built with it.
Maybe it would be helpful to include a function to explicitly disable inheritance for a specific profile?

Can someone please help me here? Thanks!

Greetings

**Comment by rasta4rest** [comment]
Exactly have the same issue here. 
not only with Jumphost. If change the default SSH Profile, either using Proxies mode all other profile saved / created previously auto matically changed following the setting of default SSH option.
